tests/test_helpers.py

```diff
@@ -341,6 +341,19 @@ def test_open_resource(self, mode):
         with app.open_resource("static/index.html", mode) as f:
             assert "<h1>Hello World!</h1>" in str(f.read())
 
+    @pytest.mark.skip(reason="this feature is not yet implemented")
+    @pytest.mark.parametrize("encoding", ("utf-8", "utf-16-le"))
+    def test_open_resource_with_encoding(self, encoding):
+        app = flask.Flask(__name__)
+        if os.path.isfile("tests/static/test.txt"):
+            os.remove("tests/static/test.txt")
+
+        with open("tests/static/test.txt", "w", encoding=encoding) as f:
+            f.write("" + "Hello World!")
```

Comment:
Is that an empty string or something non-printable/zero-width? If it's the latter, please use an escape sequence instead so it's clear it's not something useless.